roles/grubcmdline/tasks/main.yml

```diff
@@ -23,9 +22,7 @@
 - name: Set fact containing GRUB_CMDLINE_LINUX_DEFAULT
   ansible.builtin.set_fact:
     grub_cmdline_linux_default: >-
-      {{ grub_result.content | b64decode | regex_search('^GRUB_CMDLINE_LINUX_DEFAULT.*$', multiline=True)
-      | regex_replace('^GRUB_CMDLINE_LINUX_DEFAULT="(.*)"$', '\1') }}
-
+      {{ grub_result.content | b64decode | regex_search('^GRUB_CMDLINE_LINUX_DEFAULT.*$', multiline=True) | regex_replace('^GRUB_CMDLINE_LINUX_DEFAULT="(.*)"$', '\1') }} #magic___^_^___line
```

Comment:
What is `#magic___^_^___line`?

Reply:
Ah, I hadn't spotted that - it appears to be a bug with my ansible linter: https://github.com/google/yamlfmt/issues/86